### DISCUSSION TITLE
Submit button doesn't work on create Account page

### DISCUSSION THREAD

**Discussion by Bush-cat** [dani-garcia/vaultwarden#1462] [Q&A]
### Subject of the issue
Bitwarden doesn't create Account when pressing submit on the create account page. Nothing is loading, the server doesn't do anything and the page stays the same.

### Deployment environment
Ubuntu Server 20.04.2LTS running on Macbook Pro Late 2011.15 i17 2675qm; 8gb 1866MHz DDR3 SODIMM; WD Blue 2,5" 320GB HDD; faulty Rdaeon DGPU disabled

bitwarden_rs version: bitwardenrs/server:latest; Bitwarden Version 2.18.1

How the server was installed:
Docker Image with help setting up from https://theselfhostingblog.com/posts/how-to-self-host-bitwarden-on-ubuntu-server/

* Clients used: Tried with Chrome, Bitwarden Windows, Bitwarden Android

### Steps to reproduce
# start bitwarden server
sudo docker-compose up -d

open ip Adress of server in local network
press create account, fill in information
press submit

### Actual behaviour
Nothing happens

### Troubleshooting data
CONTAINER ID   IMAGE                COMMAND                  CREATED          STATUS                    PORTS                          NAMES
0a9fa0f37ca9   bitwardenrs/server   "/usr/bin/dumb-init …"   25 minutes ago   Up 25 minutes (healthy)   0.0.0.0:80->80/tcp, 3012/tcp   bitwarden_bitwarden_1

**Comment by jjlin** [comment]
You need to enable HTTPS. See https://github.com/dani-garcia/bitwarden_rs/wiki/Enabling-HTTPS.
